etl/steps/data/garden/un/2022-07-11/population.py

```diff
@@ -122,7 +122,7 @@ def _add_metric_sexratio(df: pd.DataFrame) -> pd.DataFrame:
 
 
 def _add_metric_population(df: pd.DataFrame) -> Tuple[pd.DataFrame, pd.DataFrame]:
-    df_p = df[df.metric == "population"]
+    df_p = df.loc[df.metric == "population"]
```

Comment:
Does `.loc` make any difference here?

Reply:
It does the same thing, but `.loc` works better with type hints (I'm getting warning in VSCode otherwise and wrongly inferred `pd.Series` type)

Reply:
Oh interesting. Yeah, I suppose using `.loc` is better practice than not using it, although in reality (unless doing assignments) it doesn't make a difference.